src/main/java/com/zachary_moore/lint/LintError.java

```diff
@@ -0,0 +1,63 @@
+package com.zachary_moore.lint;
+
+import java.util.Objects;
+
+
+public class LintError {
+
+  private final String jsonPath;
```

Comment:
Can we use this in the RB? We are allowing us to get this info but not using it at all in reporting